src/components/ContactCard/ContactCard.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "gatsby";
```

Comment:
Better to fix this warning.

Reply:
Can you elaborate on what needs to be done? I don't quite understand.

Reply:
Just remove this unused import. Warning should be fixed then